### DISCUSSION TITLE
I was wondering if We can already use the lib in .net 9?

### DISCUSSION THREAD

**Discussion by YBTopaz8** [taublast/DrawnUi#125] [Q&A] [answered]
Hi!
I was wondering so because whenever I add try to draw any thing, I have some issues with newly created windows in .net 9

I thought it was my project, but looks not.
I downloaded the official TitleBarDemo [here](https://github.com/dotnet/maui-samples/tree/main/9.0/UserInterface/Views/TitleBarDemo) from MS and simply added this lib and put a skialabel in a skialayout but the app kept crashing still the error;

`Microsoft.Maui.Platform.HandlerNotFoundException: 'Unable to find a IElementHandler corresponding to .Maui.Draw.SkiaLayout. register a handler for DrawnUi.Maui.Draw.SkiaLayout using Microsoft.Maui.Hosting.MauiHandlersCollectionExtensions.AddHandler or 
 Microsoft.Maui.Hosting.MauiHandlersCollectionExtensions.TryAddHandler`

(The image is from my project because it gives more details on crash, the one on a brand new sample project isn't descriptive)

Any help would be much appreciated 🙋‍♂️
![image](https://github.com/user-attachments/assets/84b76a90-a898-4954-8e95-8db00f8cf39d)

**Comment by taublast** [comment, marked as answer]
Hi,
SkiaLabel, SkiaLayout etc are virtual drawn controls to be drawn on a native view which name is `Canvas`:  
`<draw:Canvas>your skia controls</draw:Canvas>`
Please kindly show your XAML or C# UI code.
The error above would happen an all NET versions if you include a control to be used backed-up with a Handler. Only `Canvas` has handlers for normal and hardware accelerated views.
If it doesn't work we would investigate this in relation with .NET 9 then.
